<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSP_EditorClass</class>
 <widget class="QMainWindow" name="SSP_EditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSP_Editor</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow{
background:    rgb(68, 68, 68);
border: black;
}
QMenuBar{
background:   rgb(68, 68, 68);
color: white;
border: black;
}


QToolBar{
border: black;}

QTabWidget::pane { /* The tab widget frame */
    border: 1px solid black;
	background :  rgb(68, 68, 68);
	color : green;
}


QTabBar::tab{
    background:  rgb(48, 48, 48);
    border: 1px solid rgb(68, 68, 68) ;
    border-bottom-color:  rgb(68, 68, 68); /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 5px;
 	color : white;
}
QTabBar::tab:selected{
    background : rgb(111, 111, 111);
}
QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}


QTreeWidget{
background :  rgb(68, 68, 68);
spacing: 10px;
show-decoration-selected: 1; 
border:black;
 
}

QTreeWidget::Item:has-children{
spacing: 3px;
border-radius: 4px;
border: 1px solid;
color: white;
background : rgb(88, 88, 88);
}
QTreeWidget::Item:!has-children{ 
spacing: 3px;
border-radius: 1px;
border: 1px solid;
background : rgb(222, 222, 222);
}
QTreeView::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;
}



#centralWidget{
background:  rgb(68, 68, 68);
border : 1px solid black;
}

QLabel{
color : white;
}

QPushButton{
 color: white;
 background-color : rgb(111, 111, 111);
 border-radius : 2px;
padding :5px;
 border:  rgb(115, 115, 115) ;
}
QPushButton:hover{
   background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;}

QPushButton:pressed{
  background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(221, 263, 239), stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;}

</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QWidget" name="RenderWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>711</width>
        <height>571</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QWidget" name="RenderWidget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Attributes</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Asset Browser</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Files</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Scene info</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>label_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>RenderWidget</zorder>
   <zorder>RenderWidget_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionNew_scene"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_scene"/>
    <addaction name="actionSave_scene"/>
    <addaction name="separator"/>
    <addaction name="actionBuild_BPF"/>
   </widget>
   <addaction name="menuEditor"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_scene">
   <property name="text">
    <string>New scene</string>
   </property>
  </action>
  <action name="actionLoad_scene">
   <property name="text">
    <string>Load scene</string>
   </property>
  </action>
  <action name="actionSave_scene">
   <property name="text">
    <string>Save scene</string>
   </property>
  </action>
  <action name="actionBuild_BPF">
   <property name="text">
    <string>Build .BPF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SSP_Editor.qrc"/>
 </resources>
 <connections/>
</ui>
